dashboard/src/components/jobs/parameters/TimeParameters.vue

```diff
@@ -27,40 +30,37 @@
     </div>
     <div class="timefield">
       <b>Start:</b>
-      <datetime
-        @close="emitParams"
-        placeholder="Click here to set start date"
-        type="datetime"
-        v-model="start"
-        format="y-LL-dd HH:mmZZ"
-        :zone="timezone"
-        :picker-zone="timezone"
-        :max-datetime="end"
-      ></datetime>
+      <datetimefield
+        @update-datetime="setStart"
+        :timezone="timezone"
+        helpText="The value of the first timestamp in your data."
+      />
     </div>
     <div class="timefield">
       <b>End:</b>
-      <datetime
-        @close="emitParams"
-        type="datetime"
-        placeholder="Click here to set start date"
-        v-model="end"
-        :zone="timezone"
-        :picker-zone="timezone"
-        :min-datetime="start"
-        format="y-LL-dd HH:mmZZ"
-      ></datetime>
+      <datetimefield
+        @update-datetime="setEnd"
+        :timezone="timezone"
+        helpText="The end date and time of your data. This may be any between the last timestamp and the beginning of the next timestep. For example, 60 minute data with a last timestamp at 2020-12-31 23:00 should have an end between 2020-12-31 23:01 and 2021-01-01 00:00."
```

Comment:
@alorenzo175 What do you think of this end time explanation?

Reply:
I think something a little simpler like 
```suggestion
        helpText="The end date and time of your data, exclusive. For example, 60 minute data with a last timestamp at 2020-12-31 23:00 should have an end between 2020-12-31 23:01 and 2021-01-01 00:00."
```

Reply:
Working at that a bit more:

""The end date and time of your data. End is the timestamp at the beginning of the last interval. For example, 60 minute data with the last interval covering the period 2020-12-31 23:00 to 2020-12-31 23:59:59, End should have the timestamp at 2020-12-31 23:00."

Reply:
I should have left this in a single comment thread. Per Tony's previous comment:

> This is actually a little more flexible. In this example anything after 23:00 would work and 2021-01-01 00:00 would also work. Not sure of a good way to describe it though

As I understand it, the end date is acting as an exclusive bound on the timestamps and not necessarily the timespan covered by the data. I think this is primarily because these time parameters are used to generate an expected index for validating the data and that's just Pandas behavior?

Reply:
> As I understand it, the end date is acting as an exclusive bound on the timestamps and not necessarily the timespan covered by the data. I think this is primarily because these time parameters are used to generate an expected index for validating the data and that's just Pandas behavior?

Yes it is an exclusive bound. We decided that data would be assumed to be, in arbiter speak, beginning interval labelled, and it is easier to specify a time range like 2020-01-01T00:00 to 2021-01-01T00:00 than set end = 2020-12-31T23:59. Pandas is happy to include both endpoints if they fit properly in the specified frequency so there is a check to drop the last index point if it == end.

Reply:
Alright, so I've included @alorenzo175's suggestion here, which reflects the current behavior. If we need to adjust the way the end parameter is handled I think we should open an issue for that.